settings.py

```diff
@@ -24,5 +26,7 @@
 USE_TZ = False
```

Comment:
The `USE_TZ` setting is set to `False`, which means that Django will store datetimes in the database as naive datetimes. Consider setting `USE_TZ` to `True` to enable timezone-aware datetimes, which is generally recommended for most applications.